test/kuzzle/constructor.test.js

```diff
@@ -894,6 +894,53 @@ describe('Kuzzle constructor', () => {
         should(kuzzle.offlineQueue[0].query.index).be.exactly('index');
         should(kuzzle.offlineQueue[0].query.headers.authorization).be.exactly('Bearer fake-token');
       });
+
+      it('should send the checkToken request when the method is called', function () {
```

Comment:
This test should be in the `methods.test.js` file instead of the `constructor.test.js` one